### PR TITLE
Fix : Components, broken links, styling, minor bugs

### PR DESCRIPTION
**Pull request by ZuberDunge**
image

### CODE REVIEW

**Review by GuptaEesh — COMMENTED** [review]
- Good semantic HTML tags are used.
- Good use of CSS variables
Overall good to merge.

**Review by GuptaEesh — APPROVED** [review]

File: docs/password.html [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="shortcut icon" href="/images/bat-icon.png" type="image/x-icon" />
+    <link rel="stylesheet" href="/index.css" />
+  </head>
+  <body>
+    <div class="app">
+      <div class="navbar">
```
Comment (Shivanshsharma01) [inline docs/password.html:18]: It's good to wrap the main nav in a "header" tag.

File: style/profile.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
+.profile {
+    max-width: 800px;
```
Comment (Shivanshsharma01) [inline style/profile.css:2]: can use relative units here as a good practice

File: style/profile.css [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    margin: 1rem auto;
+}
+
+.profile-section-grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 2rem;
+    margin: 2rem auto;
+    padding: .5rem;
+    max-width: 1000px;
```
Comment (Shivanshsharma01) [inline style/profile.css:12]: can use relative units here too

File: docs/addresses.html [hunk 4]
```diff
@@ -57,57 +58,70 @@
...
+            <a href="/docs/category.html">
+              <div class="bat-overlay-bg">
+                <div class="bat-overlay-text pointer">Wonder Woman</div>
+              </div>
+            </a>
+          </div>
+          <div>
+            <img
+              src="https://www.denofgeek.com/wp-content/uploads/2017/11/thor-comics-reading-guide-ragnarok.jpg?fit=1024%2C544"
+              alt=""
```
Comment (Hamdrive) [inline docs/addresses.html:89]: Please make use of the alt tag

File: docs/password.html [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+          </div>
+        </div>
+      </header>
+      <!-- <div class="hero-img-cat login-section-wala">
+        <img
+          src="/images/Batman-Comics-10-Essentials-You-Need-To-Read.jpg"
+          alt=""
+        />
+        <div class="absolute-head">Welcome to BatStore</div>
+      </div> -->
```
Comment (Hamdrive) [inline docs/password.html]: Cleanup commented code

**Review by shivampandey0 — COMMENTED** [review]
Clean Code and Semantic HTML used.
Good to merge

File: docs/product.html [hunk 6]
```diff
@@ -495,12 +551,10 @@ <h1>Don’t Miss Out</h1>
...
-              <li>Return</li>
               <li>Payment</li>
               <li>Term of use</li>
-              <li>Privacy Policy</li>
-              <!-- <li>Payment options Available</li> -->
+              <li>Privacy</li>
+              <li>Payment options Available</li>
             </ul>
 
             <!-- <div>
```
Comment (Hamdrive) [inline docs/product.html]: cleanup comments here

File: index.html [hunk 7]
```diff
@@ -693,6 +699,41 @@ <h1>Don’t Miss Out</h1>
...
       </footer>
     </div>
-    <script src="./index.js"></script>
+    <script>
+      var slide = document.getElementById("slider");
+      var btn1 = document.getElementById("btn1");
+      var btn2 = document.getElementById("btn2");
+      var btn3 = document.getElementById("btn3");
+      var btn4 = document.getElementById("btn4");
+
```
Comment (Hamdrive) [inline index.html]: Create a separate JS file for your carousel, as HTML and JS shhould be in their own files

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean with good use of variables. Suggestion is to use more semantic HTML tags

**Review by Hamdrive — COMMENTED** [review]
Very good code written, great practices followed and good use of utility classes
Havee suggested to fill empty `alt` tags and cleanup commented code

File: index.html (hunk 7 above)
Comment (ZuberDunge) [reply index.html]: Updated.

File: docs/product.html (hunk 6 above)
Comment (ZuberDunge) [reply docs/product.html]: Done.

File: docs/password.html (hunk 5 above)
Comment (ZuberDunge) [reply docs/password.html]: Done

File: docs/addresses.html (hunk 4 above)
Comment (ZuberDunge) [reply docs/addresses.html:89]: Updated.

File: style/profile.css (hunk 3 above)
Comment (ZuberDunge) [reply style/profile.css:12]: Okay.

File: docs/password.html (hunk 1 above)
Comment (ZuberDunge) [reply docs/password.html:18]: okay Shivansh, I'll make changes .

**Pull request merged**